python/lsst/pipe/analysis/plotUtils.py

```diff
@@ -955,22 +955,27 @@ def plotRhoStats(axes, rhoStats):
     Parameters
     ----------
     axes : `list`
-        A list containing two `matplotlib.figure.ax` handles.
+        A list containing three `matplotlib.figure.ax` handles
     rhoStats : `dict`
         A Python dictionary object with keys 1..5, each containing a
-        `treecorr.GGCorrelation object`.
+        `treecorr.GGCorrelation` object and key 0 containing a
+        `treecorr.KKCorrelation` object.
+
+    Returns
+    -------
+        `None`
```

Comment:
Return must be named (see description and warning at: https://developer.lsst.io/python/numpydoc.html#naming-return-variables).  Also, this function doesn't have an explicit return.

Reply:
Since the return is not explicit, could I just drop the `Returns` part from the docstring?

Reply:
Yes, I think that's fine.  (We may want to eventually do a repo-wide standardizing of "to always return" or "to sometimes return", but not in this ticket!).